server/internal/billing/tracking.go

```diff
@@ -51,4 +55,5 @@ type Tracker interface {
 	TrackToolCallUsage(ctx context.Context, event ToolCallUsageEvent)
 	TrackPromptCallUsage(ctx context.Context, event PromptCallUsageEvent)
 	TrackPlatformUsage(ctx context.Context, event PlatformUsageEvent)
+	GetStoredPeriodUsage(ctx context.Context, orgID string) (*gen.PeriodUsage, error)
```

Comment:
this should be in the Repository interface, not here.

Reply:
ack